src/Driver/PhpAmqp/Driver.php

```diff
@@ -1,13 +1,12 @@
 <?php
 
-namespace Bernard\Driver;
+namespace Bernard\Driver\PhpAmqp;
```

Comment:
Wonder if simply `Amqp` would be a better name.

Reply:
Now would be indeed a good time to rename this. Will do!